docs/definitions/type-system/interface.md

```diff
@@ -1,38 +1,253 @@
 # Interface
 
-## With YAML
+An Interface is an abstract type that includes a certain set of fields that a type must include to implement the interface.
+See the [official documentation](https://graphql.org/learn/schema/#interfaces) for more details.
+
+Here is an example of an interface and two types implementing it (implementors) written with GraphQL schema language:
+```graphql
+# Character that represents any character in the Star Wars trilogy
+interface Character {
+    id: String!
+    name: String
+    friends: [Character]
+    appearsIn: [Episode]
+}
+
+type Human implements Character {
+  id: ID!
+  name: String!
+  friends: [Character]
+  appearsIn: [Episode]!
+  starships: [Starship]
+  totalCredits: Int
+}
+
+type Droid implements Character {
+  id: ID!
+  name: String!
+  friends: [Character]
+  appearsIn: [Episode]!
+  primaryFunction: String
+}
+```
+
+We'll show you how the schema above can be implemented with this bundle. There are two main ways to do 
```

Comment:
There are 4 ways to do `xml`, `graphql`, `yaml` and `annotations`. The faster way to implement a simple schema like that one is `graphql` I will document it

Reply:
Yes, thats why i wrote "the are two *main* ways".
I think we should state somewhere in the beginning, e.g. "Introduction", that there are 4 ways to define schema.